MiniSim/Menu.swift

```diff
@@ -128,10 +117,8 @@ class Menu: NSMenu {
                 continue
             }
             
-            DispatchQueue.main.async {
-                if self.items.contains(item) {
-                    item.keyEquivalent = keyEquivalent
-                }
+            if self.items.contains(item) {
```

Comment:
Here we are still missing the `DispatchQueue.main`

Reply:
I've added it back here.

Reply:
We need to check if item exists inside of the `DispatchQueue.main.async` call as it might not exists when we get to executing the inner closure on the main thread. Can you please for now bring back the old approach? And later on we can refactor it with Async/Await.